common/src/IO/MapReader.h

```diff
@@ -160,23 +136,13 @@ namespace TrenchBroom {
             void onStandardBrushFace(size_t line, Model::MapFormat targetMapFormat, const vm::vec3& point1, const vm::vec3& point2, const vm::vec3& point3, const Model::BrushFaceAttributes& attribs, ParserStatus& status) override;
             void onValveBrushFace(size_t line, Model::MapFormat targetMapFormat, const vm::vec3& point1, const vm::vec3& point2, const vm::vec3& point3, const Model::BrushFaceAttributes& attribs, const vm::vec3& texAxisX, const vm::vec3& texAxisY, ParserStatus& status) override;
         private: // helper methods
+            friend std::vector<std::unique_ptr<Model::BrushNode>> createBrushes(const vm::bbox3& worldBounds, std::vector<BrushInfo> brushInfos, ParserStatus& status);
```

Comment:
I had to look up what this `friend` does - I normally use `static` for this purpose.

According to what I read, they will have the same meaning if it's private (the only difference is, if you're calling it from outside MapReader, `static` requires calling it like `MapReader::createBrushes(...` whereas `friend` doesn't need the `MapReader::`)

Reply:
These are gone now. I had added them so I could keep the nested structs in `MapReader` private - but it's too cumbersome, so I made them public instead (in order to access them in free helper functions).